pipeline/src/metadata.ts

```diff
@@ -1,9 +1,10 @@
 import path = require("node:path");
 import fs = require("node:fs/promises");
-import { Brand } from "./table_merger";
+import { Brand } from "../../backend/schema/metadata";
```

Comment:
On a second thought, I think it will be neater for the schema to have its own package from the root directory something like `common/schema` perhaps instead of inside backend?

Reply:
I agree, I’ll do that before merging.

Reply:
Actually for simplicity given the size of this PR, I'll do it as a fast follow-up.